google-cloud-dlp/Rakefile

```diff
@@ -124,11 +75,11 @@ task :jsondoc => :yard do
     documents: [
       {
         type: "toc",
-        title: "Google::Privacy::Dlp::V2beta1::DataTypes",
+        title: "Google::Privacy::Dlp::V2::DataTypes",
```

Comment:
Do you mean to replace `V2beta1` with `V2`, or do you want to show both `V2` and `V2beta1` versions in the documentation? It looks like the `V2beta1` files are not being removed, right?